client/my-sites/plans/jetpack-plans/products-grid-i5/index.tsx

```diff
@@ -154,39 +156,39 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 										] }
 									/>
 								</li>
-							) ) }
-						</ul>
-						<div
-							className={ classNames( 'products-grid-i5__more', {
-								'is-detached': isPlanRowWrapping,
-							} ) }
-						>
-							<MoreInfoBox
-								headline={ translate( 'Need more info?' ) }
-								buttonLabel={ translate( 'Compare all product bundles' ) }
-								onButtonClick={ scrollToComparison }
-							/>
-						</div>
-					</>
-				) }
+						  ) ) }
+				</ul>
+				<div
+					className={ classNames( 'products-grid-i5__more', {
+						'is-detached': isPlanRowWrapping,
+					} ) }
+				>
+					<MoreInfoBox
+						headline={ translate( 'Need more info?' ) }
+						buttonLabel={ translate( 'Compare all product bundles' ) }
+						onButtonClick={ scrollToComparison }
+					/>
+				</div>
 			</section>
 			<section className="products-grid-i5__section">
 				<h2 className="products-grid-i5__section-title">{ translate( 'Individual Products' ) }</h2>
-				{ ! isVariationLoading && (
-					<ul className="products-grid-i5__product-grid">
-						{ sortedProducts.map( ( product ) => (
-							<li key={ product.iconSlug }>
-								<ProductCardI5
-									item={ product }
-									onClick={ onSelectProduct }
-									siteId={ siteId }
-									currencyCode={ currencyCode }
-									selectedTerm={ duration }
-								/>
-							</li>
-						) ) }
-					</ul>
-				) }
+				<ul className="products-grid-i5__product-grid">
+					{ isVariationLoading
+						? range( 3 ).map( ( n, index ) => (
```

Comment:
I wonder if it may be easier to just put `[ 1, 2, 3 ]`? `[...Array(3).keys()]` should also work! This code is fine, but I have a philosophy of reducing imports 😄

Reply:
Good point @ChaosExAnima.